bgpd/bgp_attr.h

```diff
@@ -198,6 +222,12 @@ struct attr {
 	/* MPLS label */
 	mpls_label_t label;
 
+	/* SRv6 VPN SID */
```

Comment:
Don't these belong in attr->extra?

Reply:
No. attr->extra is already abolished at current version, isn't it ?
but there belong in `struct bgp_path_info_extra`. 

that structure have same field with `struct attr` (i.e. labels.)

or Are you talking about `struct bgp_path_info_extra`'s field duplication?